### DISCUSSION TITLE
An Upcoming Change in the TinyTeX Installation Path on Windows | /en/2023/11/tinytex-path/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1539] [Comments]
Since about a month ago, I have been receiving error reports from TinyTeX users saying &ldquo;TinyTeX\bin\windows\runscript.tlu:864: no appropriate script or program found: fmtutil&rdquo; or &ldquo;I &hellip;

https://yihui.org/en/2023/11/tinytex-path/

**Comment by yoonghm** [comment]
Microsoft Windows OS does not maintain compatibility of its older standards. In Linux, Unix, macOS, users could choose to install application into `/usr/local` or `$HOME` directories. Anaconda allows users to install on any directories and uses environment variables to record the installation location. Python provides a script to return its installed location.
